src/core/include/openvino/core/graph_util.hpp

```diff
@@ -278,4 +279,10 @@ bool replace_output_update_name(Output<Node> node, const Output<Node>& node_inpu
 
 OPENVINO_API
 bool replace_node_update_name(const std::shared_ptr<Node>& target, const std::shared_ptr<Node>& replacement);
+
+OPENVINO_API
+void serialize(const std::shared_ptr<ov::Model>& m,
+               const std::string& xml_path,
+               const std::string& bin_path,
```

Comment:
Why `bin_path` is not optional? In most cases `xml_path` and `bin_path` are different in file extension only, and this is a convenient approach for OV.

Reply:
Done.